cmd/zc_enumerator.go

```diff
@@ -72,6 +82,128 @@ func newStoredObject(name string, relativePath string, lmt time.Time, size int64
 // pass each storedObject to the given objectProcessor if it passes all the filters
 type resourceTraverser interface {
 	traverse(processor objectProcessor, filters []objectFilter) error
+	isDirectory(isDest bool) bool // isDest is literally only useful on blob because we have seperate rules for blob on up/down.
```

Comment:
please clarify in the comment what's the special rule

Reply:
There actually never really was a special rule going on here. If I recall correctly, this was because of how I was trying to implement directory checks on source being a pain.